qutebrowser/mainwindow/tabwidget.py

```diff
@@ -222,32 +223,48 @@ def __init__(self, win_id, parent=None):
         config_obj = objreg.get('config')
         config_obj.changed.connect(self.set_font)
         self.vertical = False
+        self.autoHideTimer = None
```

Comment:
Please call this `auto_hide_timer` instead - camel case is only used for Qt things. Also, it probably can be `_auto_hide_timer` (i.e. "private").

Can you please also add a small description to the docstring under `Attributes:`?